### PR TITLE
TK258-1

### PR DESCRIPTION
**Pull request by karuna835**
Student in dropout functionality can be given the access back
1. provide option to enable the student back to active state
2. existing subs to be activated.

### CODE REVIEW

File: app/modules/batcave/subs/edit.php [hunk 1]
```diff
@@ -81,7 +81,8 @@ function edit_subs_status($subs) {
 
 		}
 		else if ($subs["status"] == "reinstate") {
-
+			//TK258-1 Added status pending.
+            $status = db_sanitize('pending');
```
Comment (kailasb) [inline app/modules/batcave/subs/edit.php:85]: get confirmation of the status for subs & user enrollment converted active from deleted so the subs status is active

**Review by kailasb — CHANGES_REQUESTED** [review]
Check the given feedback

File: app/modules/batcave/subs/edit.php (hunk 1 above)
Comment (karuna835) [reply app/modules/batcave/subs/edit.php:97]: Comment not required code as per discussed.

File: app/modules/batcave/subs/edit.php [hunk 2]
```diff
@@ -81,22 +81,50 @@ function edit_subs_status($subs) {
...
+				$check_access = db_query("SELECT access_type FROM user_lms_access WHERE user_id = $user_id ORDER BY id DESC LIMIT 1;");
+				if($check_access[0]['access_type'] != 'revoke')
+				{
+					$res = db_query("SELECT sis_id FROM user_enrollment WHERE user_id = $user_id AND status = 'active' ORDER BY enr_id DESC LIMIT 1;");
+				
+					db_exec("INSERT INTO user_lms_access (user_id, access_type, action_by, created) VALUES($user_id, 'revoke',".db_sanitize($_SESSION["user"]["user_id"]).",".db_sanitize(date("Y-m-d H:i:s")).");");
+
+					load_plugin("jlc");
+					$jlc = new JLC;
+					$jlc->disableAccounts([$res[0]["sis_id"]]);
```
Comment (kailasb) [inline app/modules/batcave/subs/edit.php:125]: on JLC response insert record in user_lms_access table

Comment (karuna835) [reply app/modules/batcave/subs/edit.php]: Made changes

**Review by kailasb — APPROVED** [review]
changes are proper

**Pull request merged**